### PR TITLE
Fix some grammatical errors in the README

### PR DESCRIPTION
**Pull request by SShrike**
I also hopefully clarified some of the language in your `contributing` section as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,6 @@
-
-# Swiboe
+# Swiboe ¬ [![Build Status](https://travis-ci.org/swiboe/swiboe.svg)](https://travis-ci.org/swiboe/swiboe) [![Join the chat at https://gitter.im/swiboe/swiboe](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/swiboe/swiboe?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (SirVer) [inline README.md]: The ¬ is a bit weird. It seems like it is part of the name. If you want to separate the images from the name, I'd say use - or •

**Comment by SirVer** [comment]
Much appreciated - as non native speaker I am always glad if somebody fixes my language!

One nit. otherwise lgtm.

**Comment by SShrike** [comment]
I've changed the separator between the title and the badges to a '•'.

**Comment by SirVer** [comment]
Thanks!

@homu r+

**Comment by SirVer** [comment]
seems like the tests are flaky :/. 

@homu retry

**Pull request merged**